examples/text_classification/iterable_train.py

```diff
@@ -210,15 +209,14 @@ def __iter__(self):
         if self._setup is False:
             self._setup_iterator()
             self._setup = True
-        for x in self._iterator:
-            yield x
+        yield from self._iterator
```

Comment:
Is this fix relevant to the drop of python 2 support? If not, I suggest to fix in a separate issue.

Reply:
Yes, it's using Python 3 syntax that was not possible also when supporting Python 2.